### ISSUE TITLE
On iPad dropping wave adjuster can move it slightly, making precise waves hard.

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/fourier-making-waves#170]
**Test device**
iPad
**Operating System**
iPadOS 14
**Browser**
Safari
**Problem description**
For https://github.com/phetsims/qa/issues/701. Related to https://github.com/phetsims/ratio-and-proportion/issues/371.
When dragging the wave adjusters, letting go seems to move them by a tic or so. This happens even if you lift straight up. Since the wave game needs to be exact, this can make the game fairly difficult on iPad.

**Comment by pixelzoom** [comment]
@KatieWoe what are "the wave adjusters"?  Do you mean the amplitude sliders?

**Comment by KatieWoe** [comment]
Yes. Sorry I wasn't sure of the name

**Comment by pixelzoom** [comment]
@KatieWoe if you do mean the amplitude sliders...  These are common-code sliders with custom thumbs and tracks.  So if there's a problem with the slider thumb moving after you release it, it's like a problem with common-code Slider.js.   Do you see a similar problem with other (more standard looking) sliders?

**Comment by pixelzoom** [comment]
And sorry that I can't test this myself, I don't have an iPad with me.

Do you also see this on iPhone by any chance?

**Comment by KatieWoe** [comment]
I have seen it with other sliders and even objects, but the only time I can think of where it was a potential problem was https://github.com/phetsims/ratio-and-proportion/issues/371 as mentioned above. It does happen on phone as well I believe.

**Comment by arouinfar** [comment]
This isn't a sim-specific issue. It's a general issue for sliders on touch devices. It seems like the thumb moves slightly because the user inadvertently taps the slider track while releasing the thumb. 

I played the first three levels on my phone (Google Pixel 3 XL). Occasionally, the slider thumb would snap away from the value I had set, but it was easy enough to fix. Zooming in a bit can alleviate things. Users also have the ability to type in values, so I do not think this is a blocking issue.

**Comment by pixelzoom** [comment]
Here's what I see on iPhone, and I suspect that it's the same on iPad.  When you pull your finger away from the screen, there's a tendency to do so at an angle to the screen, not perpendicular to the screen.  And this in turn cause you to pull the thumb slightly down as you release it.  On the _Wave Game_ screen...  If I keep my finger at the typical angle to the screen, I see the slider value go DOWN by a tick. If I keep my finger perpendicular to the screen, I DO NOT see this behavior.  And if I put my finger at an odd angle to screen, where I release by moving my hand "up", then I see the slider value go UP by a tick.

First, @KatieWoe can you verify this? And were you typically seeing the slider value go DOWN by a tick?

And if that's the case, there's nothing I can do about this in this sim.  As @arouinfar said in https://github.com/phetsims/fourier-making-waves/issues/170#issuecomment-910837201, this is not a sim-specific issue, and would require re-thinking Slider.js for touch devices.

**Comment by KatieWoe** [comment]
I see it jump both ways, but now that you point it out there might be a bias towards jumping down.

**Comment by pixelzoom** [comment]
This is a general "user interaction" problem. It's demonstrable with standard sliders, and occurs in other sims (see phetsims/ratio-and-proportion#371). So my recommendation is to transfer this issue to the sun repo, where Slider.js lives.   And unless the sim team feels otherwise, this issue will not be addressed for the Fourier 1.0 release. 

@arouinfar do you agree?

**Comment by arouinfar** [comment]
> @arouinfar do you agree?

Yes, please transfer this to sun. This is not a blocking issue for Fourier.
